### PR TITLE
EMSUSD-249 Refactors relative path dialog options to be more clear

### PR DESCRIPTION
**Pull request by AramAzhari-adsk**
EMSUSD-249 Refactors relative path dialog options to be more clear

### CODE REVIEW

**Review by pierrebai-adsk — CHANGES_REQUESTED** [review]
Only need to remove 3 more lines, otherwise LGTM

File: plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py [hunk 1]
```diff
@@ -142,7 +139,6 @@ def uiInit(cls, parentLayout, canBeRelative, relativeToWhat):
                 showPreviewFields = cmds.file(q=True, exists=True)
```
[the remark this answers is not in the document]
Comment (AramAzhari-adsk) [reply plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py:139]: Wouldn't we still want to show the unresolved field? (Now called Path Preview)

[the remark this answers is not in the document]
Comment (pierrebai-adsk) [reply plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py:139]: Oh sorry, the variable is also usedfor the other field, NVM, I thought it was only for the resolved path.

**Review by pierrebai-adsk — APPROVED** [review]
LGTM

**Review by samuelliu-adsk — DISMISSED** [review]

File: plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py [hunk 2]
```diff
@@ -79,23 +79,20 @@ def uiCreate(cls, parentLayout, relativeToWhat):
...
         optBoxMarginWidth = mel.eval('global int $gOptionBoxTemplateDescriptionMarginWidth; $gOptionBoxTemplateDescriptionMarginWidth += 0')
         cmds.setParent(topForm)
-        cmds.frameLayout(label=kFileOptionsStr, collapsable=False)
+        cmds.frameLayout(label=kRelativePathOptionsStr, collapsable=False)
         widgetColumn = cmds.columnLayout()
         cmds.checkBox(cls.kMakePathRelativeCheckBox, label=kMakePathRelativeStr, ann=kMakePathRelativeAnnStr)
         
         cmds.checkBox(cls.kMakePathRelativeCheckBox, edit=True, changeCommand=cls.onMakePathRelativeChanged)
         cmds.textFieldGrp(cls.kUnresolvedPathTextField, label=kUnresolvedPathStr, ann=kUnresolvedPathAnnStr, editable=False)
-        cmds.textFieldGrp(cls.kResolvedPathTextField, label=kResolvedPathStr, ann=kResolvedPathAnnStr , editable=False)
```
Comment (seando-adsk) [inline plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py:98]: I think line 39 should also be removed (where this string is set).

**Review by seando-adsk — APPROVED** [review]

**Pull request merged**